lib/logflare_web/controllers/marketing_controller.ex

```diff
@@ -30,15 +30,11 @@ defmodule LogflareWeb.MarketingController do
   end
 
   def terms(conn, _params) do
-    render(conn, "terms_of_service.html")
+    redirect(conn, external: "https://supabase.com/terms")
```

Comment:
i think need to explicitly set status to 302
```suggestion
    conn
    |> put_status(301)
    |> redirect(external: "https://supabase.com/terms")
```